pom.xml

```diff
@@ -69,6 +69,7 @@
         <module>services</module>
         <module>ui</module>
         <module>ear</module>
+        <module>tools/graph-tool</module>
```

Comment:
That is a good point. It isn't really needed in the default build. I will move it to a profile as well.